frontend/src/actions/customizationActions.tsx

```diff
@@ -0,0 +1,89 @@
+import { Dispatch } from 'redux'
+import { toast } from 'react-toastify'
+import humps from 'humps'
+
+import { Action } from '../types/action'
+import { actionCreator } from '../redux-utils/actionCreator'
+import { customizationRequest } from '../api/HttpClientInstance'
+import { errorMessage } from '../constants/errorAndSuccessMessages'
+
+// Define action types
+
+export enum CustomizationType {
```

Comment:
I think `customization` is too wide definition (we will have dashboard customization too). More over - it's just a regular user profile actions.

Reply:
changed name for `notification` as its notifications